TravisCI build sonar-maven-plugin failure due to: Failed to connect to localhost/127.0.0.1:9000: Connection refused (Connection refused)

https://github.com/KengoTODA/findbugs-slf4j/pull/72 just failed like this, but this has nothing to do with that PR (as it did not touch any POM), so this must be some build configuration issue on the current `master` which should be looked into:

```
[INFO] --- sonar-maven-plugin:3.4.1.1168:sonar (default-cli) @ findbugs-slf4j ---
[INFO] User cache: /home/travis/.sonar/cache
[ERROR] SonarQube server [http://localhost:9000] can not be reached
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] findbugs-slf4j ..................................... FAILURE [  2.142 s]
[INFO] findbugs-slf4j bug pattern ......................... SKIPPED
[INFO] findbugs-slf4j test case ........................... SKIPPED
[INFO] findbugs-slf4j sonar plugin ........................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4.456 s
[INFO] Finished at: 2018-07-23T19:10:59Z
[INFO] Final Memory: 32M/784M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.4.1.1168:sonar (default-cli) on project findbugs-slf4j: Unable to execute SonarQube: Fail to get bootstrap index from server: Failed to connect to localhost/127.0.0.1:9000: Connection refused (Connection refused) -> [Help 1]
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -20,6 +20,7 @@ script:
 jobs:
   include:
     - stage: analysis
+      if: fork = false
       script:
         - ./mvnw org.jacoco:jacoco-maven-plugin:prepare-agent verify sonar:sonar -Dgpg.skip -B
 notifications:

